src/Sdk/WebApi/WebApi/Results/ResultsHttpClient.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using GitHub.Services.Results.Contracts;
+using GitHub.Services.WebApi.Utilities.Internal;
+using System.Net.Http.Formatting;
+
+namespace GitHub.Services.Results.Client
+{
+    public class ResultsHttpClient
```

Comment:
please use `RawHttpClientBase` as the base class and extend it with anything you need.

Reply:
Great, thank you! It isn't super clear how to go about getting the token from VssOAuthCredential to be able to make calls to other services. Do you have any insight on where that would be?